crmsh/bootstrap.py

```diff
@@ -2239,6 +2236,18 @@ def setup_passwordless_with_other_nodes(init_node):
             swap_public_ssh_key(node, user)
 
 
+def sync_files_to_disk():
+    """
+    Sync file content to disk between cluster nodes
+    """
+    files_string = ""
```

Comment:
You could replace this by something like:
```
files_string = ' '.join(filter(lambda f: os.path.isfile(f), FILES_TO_SYNC))
```

Reply:
Changed
Thanks!